### PR TITLE
WIP: Add ACRE2 items

### PR DESCRIPTION
**Pull request by bumbummen99**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
This PR will properly add ACRE2 items to the Templates.

### Please specify which Issue this PR Resolves.
#1828

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### What has to be done
- [ ] Only add man packed radios and Ground Spike Antenna parts if startWithLongRangeRadio is enabled.
- [ ] Add long range radios and parts to enemy factions to be looted.

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
Please open an issue using the matching of our templates and link it to this PR.
Thank you.

**Comment by Lazejun** [comment]
There is no point in adding them to Templates,
They are already given somewhere else,
We don't need X amount of times the same items over and over and over in each Template ad its always the same

**Comment by bumbummen99** [comment]
@Lazejun Where exactly are these items being added?
The default Radio seems to convert to an 343 automatically, but the 343 only has an range of 500m in Arcarde, the other ones have 40km+. We never found any other radio ingame and i also asked if they are integrated, murphy said they are not. I could also not find them in any loottable. With this patch, they are unlocked from the start.

**Comment by Lazejun** [comment]
https://github.com/official-antistasi-community/A3-Antistasi/blob/86e49c2ee2ee109c83d98c4750c462ee742be797/A3-Antistasi/functions/init/fn_initVarServer.sqf#L700

Don't bother adding Radios to units as it probably break a lot due to lack of of Load capacity.

If you want to add them to loot make sure normal ones are added to AllRadios, and find where the Backpacks end up and add them to Allbackpacksradios

**Comment by Lazejun** [comment]
So i guess you are done with this, and you are not attempting to add ACRE items to Loot

**Review by Lazejun — CHANGES_REQUESTED** [review]
"ACRE_PRC343", //Short Range Radio
"ACRE_PRC148", //Short Range Radio
"ACRE_PRC152", //Short Range Radio
"ACRE_PRC77", //Long Range Radio
"ACRE_PRC117F" //Long Range Radio
"ACRE_SEM90", //Vehicle stuff, not a Radio, cannot be an Arsenal Item
"ACRE_VRC111", //Vehicle stuff, not a Radio, cannot be an Arsenal Item
"ACRE_VRC110", //Vehicle stuff, not a Radio, cannot be an Arsenal Item
"ACRE_VRC103", //Vehicle stuff, not a Radio, cannot be an Arsenal Item
"ACRE_VRC64", //Vehicle stuff, not a Radio, cannot be an Arsenal Item
"ACRE_SEM70", //Long Range Radio
"ACRE_SEM52SL", //Short Range Radio
"ACRE_VHF30108SPIKE", //Ground Spike
"ACRE_VHF30108MAST", //Mast
"ACRE_VHF30108" //Both Combined

The Vehicle Racks will cause Errors, these are not Items.
SR Radios in Starting Items are fine,
LRs are behind the Parameter,
The Masts and Spike could be gathered in loot to gain an Improvement.

File: A3-Antistasi/functions/init/fn_initVarServer.sqf [hunk 1]
```diff
@@ -698,6 +698,7 @@ if (A3A_hasIFA) then {
 //     ACRE ITEM MODIFICATIONS   ///
 ////////////////////////////////////
 if (A3A_hasACRE) then {initialRebelEquipment append ["ACRE_PRC343","ACRE_PRC148","ACRE_PRC152","ACRE_PRC77","ACRE_PRC117F"];};
+if (A3A_hasACRE && startWithLongRangeRadio) then {initialRebelEquipment append ["ACRE_SEM90","ACRE_VRC111","ACRE_VRC110","ACRE_VRC103","ACRE_VRC64","ACRE_SEM70","ACRE_SEM52SL","ACRE_VHF30108SPIKE","ACRE_VHF30108MAST"];};
```
Comment (Lazejun) [inline A3-Antistasi/functions/init/fn_initVarServer.sqf:701]:
```suggestion
if (A3A_hasACRE) then {initialRebelEquipment append ["ACRE_PRC343","ACRE_PRC148","ACRE_PRC152","ACRE_SEM52SL"];};
if (A3A_hasACRE && startWithLongRangeRadio) then {initialRebelEquipment append ["ACRE_SEM70", "ACRE_PRC117F", "ACRE_PRC77"];};
```

**Review by Bob-Murphy — APPROVED** [review]

**Comment by Bob-Murphy** [comment]
Bum, as you didn't respond we took over and made the changes necessary to solve this issue.
Thank you very much for your work!

**Comment by bumbummen99** [comment]
Sorry i had been a bit limited on time. Awesome, thank you for merging :)
I did want to add the spike / radios to the loot tables for vehicles / ammo boxes but i did not attempt it due to the max load thing.

**Pull request merged**